quickstatements/latest/config.json

```diff
@@ -0,0 +1,18 @@
+{
+	"site" : "${MW_SITE_NAME}" ,
+	"bot_config_file" : "/var/www/html/bot.ini" ,
+	"logfile" : "/var/log/quickstatements/tool.log" ,
+	"sites" : {
+		"${MW_SITE_NAME}" : {
+			"oauth" : { "language":"${MW_SITE_LANG}" , "project":"${MW_SITE_NAME}" , "ini_file":"/var/www/html/quickstatements/oauth.ini" , "publicMwOAuthUrl":"http://${WB_PUBLIC_HOST_AND_PORT}/w/index.php?title=Special:OAuth" , "mwOAuthUrl":"http://${WIKIBASE_HOST}/w/index.php?title=Special:OAuth" , "mwOAuthIW":"mw" } ,
+			"server" : "${WB_PUBLIC_HOST_AND_PORT}" ,
+			"api" : "http://${WIKIBASE_HOST}/w/api.php" ,
+			"pageBase" : "http://${WB_PUBLIC_HOST_AND_PORT}/wiki/" ,
+			"toolBase" : "http://${QS_PUBLIC_HOST_AND_PORT}/" ,
+			"types" : {
+				"P" : { "type":"property" , "ns":122 , "ns_prefix":"Property:" } ,
```

Comment:
these namespaces should probably be configurable at some point.
For example wikibase has NS 0 for item. and I imagine other installs do too.